example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -41,49 +44,79 @@ describe('The S3 Ingest Granules workflow', () => {
     });
   });
 
-  describe('the PostToCmr Lambda', () => {
+  describe('the MoveGranules Lambda', () => {
     let lambdaOutput;
     let files;
     const existCheck = [];
 
     beforeAll(async () => {
-      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PostToCmr');
+      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'MoveGranules');
       files = lambdaOutput.payload.granules[0].files;
       existCheck[0] = await s3ObjectExists({ Bucket: files[0].bucket, Key: files[0].filepath });
       existCheck[1] = await s3ObjectExists({ Bucket: files[1].bucket, Key: files[1].filepath });
       existCheck[2] = await s3ObjectExists({ Bucket: files[2].bucket, Key: files[2].filepath });
     });
 
     afterAll(async () => {
-      await s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].name }).promise();
-      await s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].name }).promise();
-      await s3().deleteObject({ Bucket: files[2].bucket, Key: files[2].name }).promise();
+      await s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].filepath }).promise();
+      await s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].filepath }).promise();
+      await s3().deleteObject({ Bucket: files[2].bucket, Key: files[2].filepath }).promise();
+    });
+
+    it('has a payload with updated filename', () => {
+      let i;
+      for (i = 0; i < 3; i++) {
+        expect(files[i].filename).toEqual(expectedPayload.granules[0].files[i].filename);
+      }
+    });
+
+    it('moves files to the bucket folder based on metadata', () => {
+      existCheck.forEach((check) => {
+        expect(check).toEqual(true);
+      });
+    });
+  });
+
+  describe('the PostToCmr Lambda', () => {
+    let lambdaOutput;
+    let cmrResource;
+    let cmrLink;
+
+    beforeAll(async () => {
+      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PostToCmr');
+      cmrLink = lambdaOutput.payload.granules[0].cmrLink;
+      cmrResource = await getOnlineResources(cmrLink);
     });
 
     it('has expected payload', () => {
       const granule = lambdaOutput.payload.granules[0];
       expect(granule.published).toBe(true);
       expect(granule.cmrLink.startsWith('https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=')).toBe(true);
+
+      expect(lambdaOutput.payload).toEqual(expectedPayload);
     });
 
-    it('moves files to the bucket folder based on metadata', () => {
+    it('granule is published to CMR', () => {
       const granule = lambdaOutput.payload.granules[0];
       const result = conceptExists(granule.cmrLink);
 
       expect(granule.published).toEqual(true);
       expect(result).not.toEqual(false);
+    });
 
-      existCheck.forEach((check) => {
-        expect(check).toEqual(true);
-      });
+    it('CMR resource has correct metadata', () => {
```

Comment:
I would be a little more descriptive here i.e. CMR metadata online resources contain final metadata location